src/usd/sdf_usd_parser/sensor.cc

```diff
@@ -44,11 +44,14 @@ namespace usd
 
     // TODO(adlarkin) check units to make sure they match (no documented units for SDF)
     usdCamera.CreateFocalLengthAttr().Set(
-        static_cast<float>(sdfCamera->LensFocalLength()));
+        static_cast<float>(sdfCamera->LensFocalLength() * 10));
```

Comment:
Can you add a comment that explains why you are multiplying by 10? I assume it's because SDF and USD use different units. It would be nice to have this documented in case someone else has to look at this code later.